source/ProtonPack/ProtonPack.ino

```diff
@@ -2406,7 +2308,7 @@ void cyclotron2021(int cDelay) {
           if(i_cyclotron_multiplier > 1) {
             t_cDelay = t_cDelay - i_cyclotron_multiplier;
```

Comment:
Since t_cDelay is now unsigned, I think it is possible that this might be trying to set a unsigned integer to negative, which is causing the cyclotron to not update when it should be spinning at maximum speed during firing.

Reply:
The check on line 2290, namely this:

```cpp
if(t_cDelay < 1 || t_cDelay > cDelay) {
  t_cDelay = 1;
}
```
Should account for the unsigned overflow. In the case that t_cDelay underflows, it will be caught by the second half of the 'or' statement.

You cannot feed a negative value to millisDelay, so using unsigned integers would be correct here.

Reply:
Okay, I found spots after that section which could potentially underflow (since we were setting `t_cDelay` to 1, but `i_2021_delay` is greater than 1) and fixed them. This should fix the ramp up issue.